opm/parser/eclipse/EclipseState/Schedule/Completion.cpp

```diff
@@ -212,6 +242,22 @@ namespace Opm {
         return m_wellPi;
     }
 
+    int Completion::getSegmentNumber() const {
+        return m_segment_number;
```

Comment:
If we throw an exception here, it will make the following situation a little more complicated. 
```
m_segment_number(oldCompletion->getSegmentNumber())
```
And `-1` is unusable as a segment number. How about we just return `-1` for completions of non-multi-segment wells and completions of multi-segment wells before finishing the attaching process.